src/components/Comments.js

```diff
@@ -26,4 +27,10 @@ function Comments(props) {
     </div>);
 }
 
-export default Comments;
+const mapStateToProps = function (state) {
+  return {
+    newComments: state.newComments
+  };
+};
+
+export default (Comments);
```

Comment:
You are missing an important thing that needs to happen here. You need to go back to the address book reference guide and double check that you are doing everything necessary